inst/rmd/umap_qc.rmd

```diff
@@ -0,0 +1,100 @@
+## Dimensionality Reduction 
+
+```{r message=FALSE}
+# create UMAP colored by number of genes detected 
+scater::plotUMAP(processed_sce,
+                 point_size = 0.3,
+                 point_alpha = 0.5,
+                 colour_by = "detected") +
+  scale_color_viridis_c()+
+  theme_bw() + 
+  guides(
+    color = guide_colorbar(title = "Number of \ngenes detected")
+  )
+```
+
+Compromised cells, calculated using `miQC` described above, are filtered and removed from the library prior to downstream analyses. 
+The raw counts from all remaining cells are then normalized prior to selection of highly variable genes and dimensionality reduction. 
+The above plot shows the UMAP (Uniform Manifold Approximation and Projection) embeddings for each cell, coloring each cell by the total number of genes detected per cell.
```

Comment:
I'm wondering (not for this PR, probably) if we want to use the official caption support in Rmarkdown to make these into "proper" captions. 

That would look something like this:
````
```{r message=FALSE}
#| fig.cap: UMAP plot colored by the total number of genes detected per cell.

scater::plotUMAP(processed_sce,
                 ...
````

The advantage is it would save us from referring to plots above and below.